imageprocessor/templates/index.html

```diff
@@ -1,18 +1,41 @@
 {% extends "base.html" %}
 {% block body %}
+{% load staticfiles %}
 <style>
     h2 { margin: 20px; }
     a { display: block; margin: 20px; }
 </style>
 <div>
-    <h2>Photo Tagger</h2>
-    <a href = '{% url 'classify' %}'>Click here to access Image classifier</a>
-    <a href = '{% url 'tagsearch' %}'>Click here to access tag searcher</a>
-    {% if user.is_authenticated %}
-    <a href = '{% url 'view_my_pictures' %}'>Click here to view your pictures</a>
-    {% else %}
-    <a href = '{% url 'login' %}'>Click here to login</a>
-    <a href = '{% url 'register' %}'>Click here to register</a>
-    {% endif%}
+    <h2>Welcome to Photo Tagger!</h2>
+	<!-- Icons Grid -->
+    <section class="features-icons bg-light text-center">
+      <div class="container">
+        <div class="row">
+          <div class="col-lg-3">
+              <a href = '{% url 'classify' %}' style="text-decoration: none !important;"><h3><span><img src="{% static 'img/cloud-upload.png' %}" width="100" height="100" alt=""></span> <br>Image Classifier</h3>
```

Comment:
This would be easier to read if you used indentation instead of putting a bunch of nested tags all on the same line. You also don't need the wrap the image in a h3 or span tag. Instead of having a line break via <br/> you should do this with styling.

Reply:
added indentation and corrected tags alignment.